### DISCUSSION TITLE
step-by-step description of v1.4.3 installation?

### DISCUSSION THREAD

**Discussion by weisen-p** [StephanJoubert/home_assistant_solarman#565] [Q&A]
I need to install v1.4.3 because the device serial number of my logger starts with "27". Can I somewhere find a step-by-step description how to do this installation? Help is highly appreciated. Peter.
---
Ich muss v1.4.3 installieren, weil die Seriennummer meines Loggers mit "27" beginnt. Kann ich irgendwo eine Schritt-für-Schritt-Beschreibung finden, wie ich diese Installation durchführe? Für Hilfe bin ich sehr dankbar. Peter.

**Comment by PV-Joe** [comment]
If you did not install any higher version than 1.4.3 then just follow the instruction:
https://github.com/StephanJoubert/home_assistant_solarman

If you already installed the 1.5.1 then you have to delete the complete solarman folder with file browser first.

**Reply by weisen-p** [reply]
Thank you for your reply, PV-Joe. To me it seems that the method "HACS" declared as preferred is not possible because with HACS there is no choice possible between different versions (only v1.5.1. is possible).

Is there a trick to install v1.4.3 via HACS?

**Comment by PV-Joe** [comment]
If it is not possible to choose the version 1.4.3. through the install process then delete first and do a manual installation with:
https://github.com/StephanJoubert/home_assistant_solarman/releases/tag/v1.4.3

It works for me on a Deye 8K Hybrid with LSW3 Stick Logger and a pulling interval of 20 seconds.
